google/ggrc-core
# GGRC-3487: Fix InfoPin expanding logic for objects

# Issue description
InfoPane is not expanded if user clicks on object row next to 'state-change' buttons and before 'tree-item-action' button.

# Steps to test the changes
Originally issue was opened for Tasks but is valid for other objects too.
Steps to reproduce:
1. Go to All objects page > Tasks tab
2. Hover over Task's first tier and click on it next to Start/ Finish/ Decline buttons: info pane is not expanded

**Actual Result:** Task Info pane is not expanded if click on Task first tier next to change state buttons (start, finish, decline)
**Expected Result:** Task Info pane should be expanded if click on Task first tier next to change state buttons (start, finish, decline)

# Solution description
move event handler on the DOM element that lies above in a tree.
NOTE: it was decided not to remove logic for "itemSelector" attribute and leave it in more flexible way.

# Sanity checklist


# PR Review checklist
